content/docs/3.0/tasks/best-practices/pd-scheduling.md

```diff
@@ -0,0 +1,268 @@
+
+## Query scheduling status
+
+You can check the status of scheduling system through metrics, [pd-ctl](../../../reference/tools/pd-ctl) and logs. This section briefly introduces the methods of metrics and pd-ctl.
+
+### Operator status
+
+The **Grafana PD/Operator** page shows the metrics about operators, among which:
+
+- Schedule operator create: Operator creating information
+- Operator finish duration: Execution time consumed by each operator
+- Operator step duration: Execution time consumed by the operator step
+
+You can query operators using pd-ctl with the following commands:
+
+- `operator show`: Queries all operators generated in the current scheduling task
+- `operator show [admin | leader | region]`: Queries operators by type
+
+### Balance status
+
+The **Grafana PD/Statistics - Balance** page shows the metrics about load balancing, among which:
+
+- Store leader/region score: Score of each store
+- Store leader/region count: The number of leaders/regions in each store
+- Store available: Available storage on each store
+
+You can use store commands of pd-ctl to query the balance status of each store.
+
+### Hot Region status
+
+The **Grafana PD/Statistics - hotspot** page shows the metrics about hot regions, among which:
+
+- Hot write region’s leader/peer distribution: the leader/peer distribution in hot write regions
+- Hot read region’s leader distribution: the leader distribution in hot read regions
+
+You can also query the status of hot regions using pd-ctl with the following commands:
+
+- `hot read`: Queries hot read regions
+- `hot write`: Queries hot write regions
+- `hot store`: Queries the distribution of hot regions by store
+- `region topread [limit]`: Queries the region with top read traffic
+- `region topwrite [limit]`: Queries the region with top write traffic
+
+### Region health
+
+The **Grafana PD/Cluster/Region health** panel shows the metrics about regions in abnormal states.
+
+You can query the list of regions in abnormal states using pd-ctl with region check commands:
+
+- `region check miss-peer`: Queries regions without enough peers
+- `region check extra-peer`: Queries regions with extra peers
+- `region check down-peer`: Queries regions with down peers
+- `region check pending-peer`: Queries regions with pending peers
+
+## Control scheduling strategy
+
+You can use pd-ctl to adjust the scheduling strategy from the following three aspects.
+
+### Add/delete scheduler manually
+
+PD supports dynamically adding and removing schedulers directly through pd-ctl. For example:
+
+- `scheduler show`: Shows currently running schedulers in the system
+- `scheduler remove balance-leader-scheduler`: Removes (disable) balance-leader-scheduler
+- `scheduler add evict-leader-scheduler-1`: Adds a scheduler to remove all leaders in Store 1
```

Comment:
Can users use something other than `1`?

Reply:
Really appreciate your suggestions. I'll work on it.

Reply:
Hey Ana, here I think we can make it consistent by using `scheduler remove/add [scheduler name]`, but I am not sure with "[operator [add | remove]](https://tikv.org/docs/3.0/reference/tools/pd-ctl/#operator-show-add-remove)". The examples are concrete, yes, but more accessible to users. What's your opinion?